<?xml version="1.0"?>
<launch>
    <arg name="record_topics" default="joint_states"/>
    <arg name="log_directory" default="data/robot"/>

    <!-- Setup the node with parameters for what we want to listen to -->
    <node
        name="data_logger_bag"
        pkg="data_logger_bag"
        type="logger.py"
        respawn="true"
        output="screen">

        <!-- This location assumes that it is in your home dir. NOTE: path will be created for you -->
        <!-- default: data -->
        <param name="log_directory" value="$(arg log_directory)" type="string"/>

        <!-- Note: this is a space-delimited list -->
        <param name="record_topics" value="$(arg record_topics)" type="string"/>
    </node>
</launch>
